bin/action.min.js

```diff
@@ -11232,6 +11232,15 @@ async function deployPreview(gacFilename, deployConfig) {
   const deploymentResult = JSON.parse(deploymentText.trim());
   return deploymentResult;
 }
+async function removePreview(gacFilename, deployConfig) {
+  const {
+    projectId,
+    channelId
+  } = deployConfig;
+  const removeDeployment = await execWithCredentials(["hosting:channel:delete", channelId, "--force"], projectId, gacFilename);
```

Comment:
What if the `channelId` is autogenerated, and not set manually? When so, we can't know the channelId, which means we cannot delete it.
IMO, it makes sense to add a check that `removeOnClose` can only be used when `channelId` is set explicitly.

Reply:
I think we have a function called `getChannelID` that detects that and returns the correct channelID whether it's manually set or auto-generated.

Reply:
`  const channelId = getChannelId(configuredChannelId, context); ` exists to get the channelID somewhere in the `index.ts`file